save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/storage/DefaultStorageProjectReactor.kt

```diff
@@ -188,7 +191,18 @@ class DefaultStorageProjectReactor<K : Any>(
                 { function(this) }.toMono()
             }
 
-    private fun Pair<CompletedPart, Long>.completedPart() = first
+    companion object {
+        private const val MULTI_PART_UPLOAD_MIN_PART_SIZE_IN_BYTES = 5 * 1024 * 1024
```

Comment:
please add comments, or link to docs/stackoverflow about this
I beleive we will forget about it very soon without explanation

Reply:
added link to S3 limits + supported abort operation for case when we fail to upload some part